tests/generation/Bears.py

```diff
@@ -107,14 +106,14 @@
 
 class TestBears(unittest.TestCase):
 
-    def setUp(self):
+    def setUp(self, log_printer=None):
```

Comment:
Ah, it's not required for tests because log_printer isn't required to be passed `setUp` unlike the previous `coalib` methods. So `self.log_printer = None` should work just fine here.